src/test/java/io/burt/athena/support/TestDelayedCompletableFuture.java

```diff
@@ -1,18 +1,38 @@
 package io.burt.athena.support;
 
+import org.mockito.AdditionalAnswers;
+
 import java.time.Duration;
 import java.time.Instant;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+
 public class TestDelayedCompletableFuture<T> extends CompletableFuture<T> {
     public CompletableFuture<T> wrappedFuture;
     public Instant finishedAt;
     public TestClock clock;
 
-    public TestDelayedCompletableFuture(CompletableFuture<T> wrappedFuture, Duration delay, TestClock clock) {
+    static <T> CompletableFuture<T> create(CompletableFuture<T> future, Duration delay, TestClock clock) {
```

Comment:
With this package private, is there any reason the whole class shouldn't be package private?

Reply:
I decided to make it public instead, to conform with the existing support classes.